render/detailed/summary.go

```diff
@@ -344,13 +344,14 @@ func groupNodeSummary(base NodeSummary, r report.Report, n report.Node) (NodeSum
 }
 
 func pluralize(counters report.Counters, key, singular, plural string) string {
-	if c, ok := counters.Lookup(key); ok {
-		if c == 1 {
-			return fmt.Sprintf("%d %s", c, singular)
-		}
-		return fmt.Sprintf("%d %s", c, plural)
+	c, ok := counters.Lookup(key)
+	if !ok {
+		c = 0
```

Comment:
Force of habit, I tend to assume the value is 'some undefined value' when err is not nil. Of course, this is ok, not err, so slightly different conventions, but I'd still rather keep the explicit zero value to make it obvious what's going on.